lizard_auth_server/models.py

```diff
@@ -60,14 +100,14 @@ class Portal(models.Model):
         verbose_name=_('shared secret'),
         max_length=64,
         unique=True,
-        default=gen_key('Portal', 'sso_secret'),
+        default=GenKey('Portal', 'sso_secret'),
```

Comment:
Misschien kunnen we dit simpeler doen. Is er een default nodig? Waarom ik het vraag is omdat het volgens mij in de `save()` ook aangeroepen wordt. Dus als je een Portal aanmaakt, dan de key/secret kopieert, en daarna savet, dan is de key/secret opeens anders. (=Bug)

Dus... volgens mij zou het moeten werken om de default leeg te laten en alleen op de save() te vertrouwen. Tenminste... als m'n geheugen klopt.

Kan gelijk "editable=False" op het veld gezet worden, misschien.

En is de serializable versie van GenKey dan nog nodig?